transaction-status/src/lib.rs

```diff
@@ -23,7 +23,7 @@ pub enum RpcInstruction {
     Parsed(Value),
 }
 
-/// A duplicate representation of a Message for pretty JSON serialization
+/// A duplicate representation of a CompiledInstruction for pretty JSON serialization
 #[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
 #[serde(rename_all = "camelCase")]
 pub struct RpcCompiledInstruction {
```

Comment:
In a follow-up, it'd be nice to rename this to `UiCompiledInstruction` too.   The Rpc* types ideally now only live in rpc*.rs files

Reply:
Sounds good! I'll handle transaction-status in a separate pr